tests/foreman/api/test_http_proxy.py

```diff
@@ -263,3 +263,49 @@ def test_positive_assign_http_proxy_to_products():
         assert r.http_proxy_id == http_proxy_b.id
 
     product_a.sync({'async': True})
+
+
+@pytest.mark.tier2
+def test_positive_sync_proxy_with_certificate(target_sat):
+    """Assign http_proxy with ca.crt to Repo and test
+     whether http_proxy and cert are used during sync.
+
+       Steps:
+       - Generate cert files with custom_cert_generate,
+         collect cacert path.
+       - Create new org, product, and http-proxy with cacert.
+       - Create repo using selected http proxy, perform repo sync.
+
+    :expectedresults: http-proxy with cacert is assigned to repo,
+        sync operation uses assigned http-proxy with the cert.
+
+    """
```

Comment:
```suggestion
    """Assign http_proxy with ca.crt to Repo and test
     whether http_proxy and cert are used during sync.

       :steps:
       1. Generate cert files with custom_cert_generate,
         collect cacert path.
       2. Create new org, product, and http-proxy with cacert.
       3. Create repo using selected http proxy, perform repo sync.

    :expectedresults: http-proxy with cacert is assigned to repo,
        sync operation uses assigned http-proxy with the cert.
    """
```
Also if this is related to a customer case add the `:BZ:` and `:customerscenario:` tags.

Reply:
Yep those will be in the next commit